cmake/configs/nuttx_px4fmu-v2_lpe.cmake

```diff
@@ -3,3 +3,8 @@ include(cmake/configs/nuttx_px4fmu-v2_default.cmake)
 list(APPEND config_module_list
 	modules/local_position_estimator
 	)
+
+list(REMOVE_ITEM config_module_list
```

Comment:
Is this because we are out of memory with LPE? It would be nice to leave in for people to compare both during the transition from INAV to LPE.